geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/IteratorStackCounter.scala

```diff
@@ -0,0 +1,39 @@
+/***********************************************************************
+  * Copyright (c) 2013-2015 Commonwealth Computer Research, Inc.
+  * All rights reserved. This program and the accompanying materials
+  * are made available under the terms of the Apache License, Version 2.0 which
+  * accompanies this distribution and is available at
+  * http://www.opensource.org/licenses/apache2.0.php.
+  *************************************************************************/
+
+package org.locationtech.geomesa.utils.stats
+
+import org.opengis.feature.simple.SimpleFeature
+
+/**
+ * The IteratorStackCounter keeps track of the number of Accumulo iterators
+ * that are instantiated as a result of a query
+ *
+ * @param count number of iterators
+ */
+class IteratorStackCounter(var count: Long = 1) extends Stat {
+  override def observe(sf: SimpleFeature): Unit = { }
+
+  override def add(other: Stat): Stat = {
+    other match {
+      case o: IteratorStackCounter => count += o.count
+    }
+    this
+  }
+
+  override def toJson(): String = s"""{ "count": $count }"""
+
+  override def clear(): Unit = count = 1L
```

Comment:
wouldn't the count default to 0? although maybe since we only would hit this in an iterator, there is always 1?